### ISSUE TITLE
Gene mapping database issues

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/BridgeDb#64]
Several things on the gene mapping database:

1. Last version is from Sept. 2016, so a new version should be created (regularly).
2. On the website of BridgeDB, the downloading of the mapping databases for metabolites and genes should have the same layout; the current layout is very confusing for (new) users.
3. The creation of the gene-mapping-file is not mentioned in any of the repositories here; this should be added just like the metabolite mapping file.

@JonathanMELIUS @egonw

**Comment by egonw** [comment]
@DeniseSl22, is this still relevant?

**Comment by DeniseSl22** [comment]
@egonw : yes, the GeneProtein repo doesn't seem to be up to date with the current code; last commit was end of 2021 (I made one for the .cff file, but that was not on content ). @tabbassidaloii : is this the current repo to create mapping files for genes and proteins? https://github.com/bridgedb/create-bridgedb-genedb

**Comment by tabbassidaloii** [comment]
Yes, it is. I am using the scripts here to update the databases. Though some minor changes are required.
A new README.md file will be added with some more details.

**Comment by DeniseSl22** [comment]
@tabbassidaloii : thank you for the quick response! I don't think a new readme file is necessary, but an updated one would be nice. So, if these repo's are up to date, might I suggest @egonw to add them to the download page? https://bridgedb.github.io/data/gene_database/ Just that everyone knows which code is used to create the mapping files.

**Comment by tabbassidaloii** [comment]
No, thank you! :)
Nice, indeed. I will push the minor changes by tomorrow so the repo would be up-to-date.
